src/requirements.txt

```diff
@@ -1 +1,19 @@
-# None, yet.
+playwright==1.49.0
+pluggy==1.5.0
+pyee==12.0.0
+PySocks==1.7.1
+pytest==8.3.3
+pytest-base-url==2.1.0
+pytest-playwright==0.6.2
+python-slugify==8.0.4
+requests==2.32.3
+sniffio==1.3.1
+sortedcontainers==2.4.0
+text-unidecode==1.3
+tomli==2.2.1
+trio==0.27.0
+trio-websocket==0.11.1
+typing_extensions==4.12.2
+urllib3==2.2.3
+websocket-client==1.8.0
+wsproto==1.2.0
```

Comment:
Of this list, I think we need to specify only `playwright`.  (The process of installing that will install _its_ dependencies, so we don't need to list them here.)

Also, I think we need `PyYAML`, which isn't listed here.